tests/console/rsync.pm

```diff
@@ -18,6 +18,12 @@ use utils;
 use version_utils qw(is_opensuse is_sle is_jeos);
 
 sub run {
+    # try to install rsync if the test does not run on JeOS
+    if (!is_jeos) {
+        select_console 'root-console';
+        zypper_call('-t in rsync', dumb_term => 1);
```

Comment:
What does `-t` mean? Can't find it in zypper global options

Reply:
"-t, --type type
Type of package (default: package). See section Package Types for list of available package types."

From what I understand it's useful so that only packages are installed and patches for example are skipped, but I might be wrong. 

As far as this test goes it works without it too, but I thought its safer to have it.

Reply:
I thought just having -t will imply the default type. Terse makes more sense tough.

Reply:
Default value is used without -t. You can try with `lamp_server` pattern for example:
```bash
zypper in lamp_server #error, did not find package
zypper in -t pattern lamp_server  #works
```

Reply:
If it takes the default without it I would leave it like this. I will check the difference pattern makes tough.